tsconfig.json

```diff
@@ -3,6 +3,7 @@
     "outDir": "./dist/",
     "module": "commonjs",
     "esModuleInterop": true,
+    "resolveJsonModule": true,
```

Comment:
This allows us to import json files directly without having to do all the `readFileSync` stuff. This way gives us type checking with intellisense too